### PR TITLE
Require component rendered as child of `Link` to pass event to `onClick` handler

### PR DESCRIPTION
**Pull request by pie6k**
Currently, if you render `next/link` like

```ts
<Link>
  <CustomComponent />
</Link>
```

and have 

```tsx
interface Props {
  onClick?: () => void; // <— note we're not passing event as an argument
}

function CustomComponent({ onClick }: Props) {
  return <div onClick={() => onClick?.()}>Hello</div>
}
```

It'll result in error

```
link.js?f421:21 Uncaught TypeError: Cannot read property 'defaultPrevented' of undefined
```

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes

### CODE REVIEW

File: packages/next/client/link.tsx [hunk 1]
```diff
@@ -282,6 +282,11 @@ function Link(props: React.PropsWithChildren<LinkProps>) {
   } = {
     ref: setRef,
     onClick: (e: React.MouseEvent) => {
+      if (!e) {
+        if (process.env.NODE_ENV !== 'production') {
```
Comment (styfle) [inline packages/next/client/link.tsx:286]: I think these `if` statements should be swapped so that the whole thing can be removed in production

**Review by styfle — COMMENTED** [review]
We need an integration test to confirm this is working as expected

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 16.2s | 16.2s | ⚠️ +21ms |
| buildDurationCached | 3.8s | 3.7s | -84ms |
| nodeModulesSize | 50.2 MB | 50.2 MB | ⚠️ +583 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.869 | 2.883 | ⚠️ +0.01 |
| / avg req/sec | 871.3 | 867.24 | ⚠️ -4.06 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.868 | 1.891 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 1338.28 | 1321.93 | ⚠️ -16.35 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 23.1 kB | 23.1 kB | ✓ |
| webpack-HASH.js gzip | 1.5 kB | 1.5 kB | ✓ |
| Overall change | 67 kB | 67 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _app-HASH.js gzip | 980 B | 980 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 904 B | 904 B | ✓ |
| image-HASH.js gzip | 4.12 kB | 4.12 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 12.8 kB | 12.8 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 490 B | 490 B | ✓ |
| Overall change | 490 B | 490 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 531 B | 531 B | ✓ |
| link.html gzip | 542 B | 542 B | ✓ |
| withRouter.html gzip | 523 B | 523 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>_buildManifest.js</strong></summary>

```diff
@@ -19,7 +19,7 @@ self.__BUILD_MANIFEST = {
   "/image": [
     "static\u002Fchunks\u002Fpages\u002Fimage-ffe48e73c00abc6054aa.js"
   ],
-  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-7a3c1e637e71f2a0e489.js"],
+  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-93cea39aa5eac76ef206.js"],
   "/routerDirect": [
     "static\u002Fchunks\u002Fpages\u002FrouterDirect-e5f4a33d4c04b51b13bc.js"
   ],
```
</details>
<details>
<summary>Diff for <strong>link-HASH.js</strong></summary>

```diff
@@ -199,6 +199,11 @@
         var childProps = {
           ref: setRef,
           onClick: function onClick(e) {
+            if (!e) {
+              if (false) {
+              }
+            }
+
             if (child.props && typeof child.props.onClick === "function") {
               child.props.onClick(e);
             }
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -27,7 +27,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/pages/link-7a3c1e637e71f2a0e489.js"
+      src="/_next/static/chunks/pages/link-93cea39aa5eac76ef206.js"
       defer=""
     ></script>
     <script src="/_next/static/BUILD_ID/_buildManifest.js" defer=""></script>
```
</details>
</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.2s | 13.4s | ⚠️ +236ms |
| buildDurationCached | 5.2s | 5.3s | ⚠️ +60ms |
| nodeModulesSize | 50.2 MB | 50.2 MB | ⚠️ +583 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.919 | 3.207 | ⚠️ +0.29 |
| / avg req/sec | 856.32 | 779.54 | ⚠️ -76.78 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.876 | 1.962 | ⚠️ +0.09 |
| /error-in-render avg req/sec | 1332.38 | 1274.43 | ⚠️ -57.95 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| 17.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 14 kB | 14 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 10.6 kB | 10.6 kB | ✓ |
| webpack-HASH.js gzip | 1.19 kB | 1.19 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _app-HASH.js gzip | 965 B | 965 B | ✓ |
| _error-HASH.js gzip | 3.71 kB | 3.71 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 2.94 kB | 2.94 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.7 kB | 17.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 498 B | 499 B | ⚠️ +1 B |
| Overall change | 498 B | 499 B | ⚠️ +1 B |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 576 B | 576 B | ✓ |
| link.html gzip | 589 B | 589 B | ✓ |
| withRouter.html gzip | 570 B | 570 B | ✓ |
| Overall change | 1.74 kB | 1.74 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>_buildManifest.js</strong></summary>

```diff
@@ -16,7 +16,7 @@ self.__BUILD_MANIFEST = {
   "/hooks": [
     "static\u002Fchunks\u002Fpages\u002Fhooks-f3d7a61f2e1bf307c8a9.js"
   ],
-  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-2bc441b40fdd53a8f46a.js"],
+  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-69af5ceefcdae3d120f4.js"],
   "/routerDirect": [
     "static\u002Fchunks\u002Fpages\u002FrouterDirect-6969fe7df87ea0f513b7.js"
   ],
```
</details>
<details>
<summary>Diff for <strong>link-HASH.js</strong></summary>

```diff
@@ -265,6 +265,11 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
         var childProps = {
           ref: setRef,
           onClick: function onClick(e) {
+            if (!e) {
+              if (false) {
+              }
+            }
+
             if (child.props && typeof child.props.onClick === "function") {
               child.props.onClick(e);
             }
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -31,7 +31,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/pages/link-2bc441b40fdd53a8f46a.js"
+      src="/_next/static/chunks/pages/link-69af5ceefcdae3d120f4.js"
       defer=""
     ></script>
     <script src="/_next/static/BUILD_ID/_buildManifest.js" defer=""></script>
```
</details>
</details>
Commit: bb9a6877ad44f5ee4956e8c434b3b62bfc01d6b9

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 19.8s | 19.7s | -94ms |
| buildDurationCached | 7.5s | 7.4s | -122ms |
| nodeModulesSize | 359 MB | 359 MB | ⚠️ +385 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 4.403 | 4.344 | -0.06 |
| / avg req/sec | 567.75 | 575.49 | +7.74 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 2.065 | 1.997 | -0.07 |
| /error-in-render avg req/sec | 1210.82 | 1252.01 | +41.19 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| 450.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42 kB | 42 kB | ✓ |
| main-HASH.js gzip | 27.4 kB | 27.4 kB | ✓ |
| webpack-HASH.js gzip | 1.44 kB | 1.44 kB | ✓ |
| Overall change | 71 kB | 71 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _app-HASH.js gzip | 1.36 kB | 1.36 kB | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 326 B | 326 B | ✓ |
| dynamic-HASH.js gzip | 2.57 kB | 2.57 kB | ✓ |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 919 B | 919 B | ✓ |
| image-HASH.js gzip | 4.94 kB | 4.94 kB | ✓ |
| index-HASH.js gzip | 263 B | 263 B | ✓ |
| link-HASH.js gzip | 2.2 kB | 2.2 kB | ✓ |
| routerDirect..HASH.js gzip | 321 B | 321 B | ✓ |
| script-HASH.js gzip | 383 B | 383 B | ✓ |
| withRouter-HASH.js gzip | 318 B | 318 B | ✓ |
| 85e02e95b279..7e3.css gzip | 107 B | 107 B | ✓ |
| Overall change | 14.6 kB | 14.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 460 B | 460 B | ✓ |
| Overall change | 460 B | 460 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 531 B | 531 B | ✓ |
| link.html gzip | 545 B | 545 B | ✓ |
| withRouter.html gzip | 526 B | 526 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>_buildManifest.js</strong></summary>

```diff
@@ -13,7 +13,7 @@ self.__BUILD_MANIFEST = {
   "/head": ["static\u002Fchunks\u002Fpages\u002Fhead-7100d3b2a548f0e4.js"],
   "/hooks": ["static\u002Fchunks\u002Fpages\u002Fhooks-538d621a0e670391.js"],
   "/image": ["static\u002Fchunks\u002Fpages\u002Fimage-f97943edf7ae3dd3.js"],
-  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-27386a147a7100fa.js"],
+  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-34eb165692af8fd9.js"],
   "/routerDirect": [
     "static\u002Fchunks\u002Fpages\u002FrouterDirect-76232dd6bc335a24.js"
   ],
```
</details>
<details>
<summary>Diff for <strong>link-HASH.js</strong></summary>

```diff
@@ -260,6 +260,8 @@
         var childProps = {
           ref: setRef,
           onClick: function(e) {
+            if (false) {
+            }
             if (child.props && typeof child.props.onClick === "function") {
               child.props.onClick(e);
             }
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -27,7 +27,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/pages/link-27386a147a7100fa.js"
+      src="/_next/static/chunks/pages/link-34eb165692af8fd9.js"
       defer=""
     ></script>
     <script src="/_next/static/BUILD_ID/_buildManifest.js" defer=""></script>
```
</details>
</details>
<hr/>
<details>
<summary><strong>Default Build with SWC</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 23.9s | 24.2s | ⚠️ +299ms |
| buildDurationCached | 7.4s | 7.4s | ⚠️ +5ms |
| nodeModulesSize | 359 MB | 359 MB | ⚠️ +385 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 4.414 | 4.344 | -0.07 |
| / avg req/sec | 566.33 | 575.5 | +9.17 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 2.024 | 2.03 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1235.29 | 1231.44 | ⚠️ -3.85 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| 450.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.1 kB | 42.1 kB | ✓ |
| main-HASH.js gzip | 27.4 kB | 27.4 kB | ✓ |
| webpack-HASH.js gzip | 1.44 kB | 1.44 kB | ✓ |
| Overall change | 71.2 kB | 71.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _app-HASH.js gzip | 1.35 kB | 1.35 kB | ✓ |
| _error-HASH.js gzip | 180 B | 180 B | ✓ |
| amp-HASH.js gzip | 305 B | 305 B | ✓ |
| css-HASH.js gzip | 321 B | 321 B | ✓ |
| dynamic-HASH.js gzip | 2.56 kB | 2.56 kB | ✓ |
| head-HASH.js gzip | 342 B | 342 B | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| image-HASH.js gzip | 4.97 kB | 4.97 kB | ✓ |
| index-HASH.js gzip | 256 B | 256 B | ✓ |
| link-HASH.js gzip | 2.22 kB | 2.22 kB | ✓ |
| routerDirect..HASH.js gzip | 314 B | 314 B | ✓ |
| script-HASH.js gzip | 375 B | 375 B | ✓ |
| withRouter-HASH.js gzip | 309 B | 309 B | ✓ |
| 85e02e95b279..7e3.css gzip | 107 B | 107 B | ✓ |
| Overall change | 14.5 kB | 14.5 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 459 B | 459 B | ✓ |
| Overall change | 459 B | 459 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | pie6k/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 534 B | 534 B | ✓ |
| link.html gzip | 547 B | 547 B | ✓ |
| withRouter.html gzip | 528 B | 528 B | ✓ |
| Overall change | 1.61 kB | 1.61 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>_buildManifest.js</strong></summary>

```diff
@@ -13,7 +13,7 @@ self.__BUILD_MANIFEST = {
   "/head": ["static\u002Fchunks\u002Fpages\u002Fhead-7100d3b2a548f0e4.js"],
   "/hooks": ["static\u002Fchunks\u002Fpages\u002Fhooks-538d621a0e670391.js"],
   "/image": ["static\u002Fchunks\u002Fpages\u002Fimage-f97943edf7ae3dd3.js"],
-  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-27386a147a7100fa.js"],
+  "/link": ["static\u002Fchunks\u002Fpages\u002Flink-34eb165692af8fd9.js"],
   "/routerDirect": [
     "static\u002Fchunks\u002Fpages\u002FrouterDirect-76232dd6bc335a24.js"
   ],
```
</details>
<details>
<summary>Diff for <strong>link-HASH.js</strong></summary>

```diff
@@ -260,6 +260,8 @@
         var childProps = {
           ref: setRef,
           onClick: function(e) {
+            if (false) {
+            }
             if (child.props && typeof child.props.onClick === "function") {
               child.props.onClick(e);
             }
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -27,7 +27,7 @@
       defer=""
     ></script>
     <script
-      src="/_next/static/chunks/pages/link-27386a147a7100fa.js"
+      src="/_next/static/chunks/pages/link-34eb165692af8fd9.js"
       defer=""
     ></script>
     <script src="/_next/static/BUILD_ID/_buildManifest.js" defer=""></script>
```
</details>
</details>
Commit: cf44f9b2ab77ee61d6169d84613f6f14b1bc8a36

**Review by ijjk — APPROVED** [review]
Thanks for the PR!

**Pull request merged**